src/taco-cli/cli/kit.ts

```diff
@@ -95,8 +104,9 @@ class Kit extends commands.TacoCommandBase {
         return Kit.promptUser(resources.getString("CommandKitSelectProjectUpdatePrompt"))
         .then(function (answer: string): Q.Promise<any> {
             if (answer && answer.length > 0 ) {
-                answer = answer.toLowerCase();
-                if (resources.getString("PromptResponseYes").split("\n").indexOf(answer) !== -1) {
+                // Try the lower-case version of the answer as well (This is needed so we can have better test coverage for "yes" and "no" response code paths)
```

Comment:
Ah, I see. In that case I'd be inclined to call toLowerCase on the getString() response as well, to make sure we are comparing both sides as lower case. I see that as less of a "making allowances for test scenarios" and more "protecting us against small mistakes in (possibly localized) strings", which also happens to mean we can work in a test scenario